src/branch-manager-service/shared/firestore-models/pull-request.ts

```diff
@@ -12,7 +12,7 @@ import {firestoreInstance} from '../firebase-common';
 export interface BranchManagerPullRequest {
   labels: string[];
   latestCommitSha: string;
-  org: string;
+  owner: string;
```

Comment:
Add description for `owner` since it's not as obvious as `org`?

Reply:
While it isn't as obvious because we haven't been using it. I don't feel like it warrants an explanation as it is the naming that Github uses with its API.